client/src/Components/RelatedContainer/RelatedProducts/StarButton/StarButton.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { useState } from "react"
+import ProductComparison from "../ProductComparison/ProductComparison.jsx"
+import StarIcon from "./StarIcon.jsx"
+
+const StarButton = ({ thisProduct, handleClick }) => {
+  const [displayComparison, setDisplayComparison] = useState(false)
+
+  const togglePopup = () => {
+    setDisplayComparison((prevBool) => !prevBool)
+  }
+
+  return (
+    <React.Fragment>
```

Comment:
nothing major but I think you can substitute <React.Fragment> for <> and </>

Reply:
👍 okay, I'll update that